experiment/build/gcb_build.py

```diff
@@ -39,84 +40,76 @@
 logger = logs.Logger('builder')  # pylint: disable=invalid-name
 
 
+def _get_buildable_images(fuzzer=None, benchmark=None):
+    return docker_images.get_images_to_build([fuzzer], [benchmark])
+
+
 def build_base_images():
     """Build base images on GCB."""
-    _build(get_build_config_file('base-images.yaml'), 'base-images')
+    image_templates = {'base-image': _get_buildable_images()['base-image']}
+    config = generate_cloudbuild.create_cloud_build_spec(image_templates,
+                                                         build_base_images=True)
+    _build(config, 'base-images')
 
 
 def build_coverage(benchmark):
     """Build coverage image for benchmark on GCB."""
-    coverage_binaries_dir = exp_path.filestore(
-        build_utils.get_coverage_binaries_dir())
-    substitutions = {
-        '_GCS_COVERAGE_BINARIES_DIR': coverage_binaries_dir,
-        '_BENCHMARK': benchmark,
+    buildable_images = _get_buildable_images(benchmark=benchmark)
+    image_templates = {
+        image_name: image_spec
+        for image_name, image_spec in buildable_images.items()
+        if 'coverage' in image_name
     }
-    config_file = get_build_config_file('coverage.yaml')
+    config = generate_cloudbuild.create_cloud_build_spec(image_templates,
+                                                         benchmark=benchmark)
     config_name = 'benchmark-{benchmark}-coverage'.format(benchmark=benchmark)
-    _build(config_file, config_name, substitutions)
+    _build(config, config_name)
 
 
-def _build(config_file: str,
+def _build(config: Dict,
            config_name: str,
-           substitutions: Dict[str, str] = None,
            timeout_seconds: int = GCB_BUILD_TIMEOUT
           ) -> new_process.ProcessResult:
     """Build each of |args| on gcb."""
```

Comment:
Not yoru fault but this comment is stale, could you update it please?

Reply:
Noted